Directory.Packages.props

```diff
@@ -1,40 +1,40 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
what was changed here?

Reply:
Agreed; In generanl I'd prefer more PRs instead of doing a lot in one PR; easier to review and results in a meaner CHANGELOG.md for the next release, too :D